server/src/main/java/org/elasticsearch/action/synonyms/TransportPutSynonymsAction.java

```diff
@@ -29,11 +29,11 @@ public TransportPutSynonymsAction(TransportService transportService, ActionFilte
     }
 
     @Override
-    protected void doExecute(Task task, PutSynonymsAction.Request request, ActionListener<PutSynonymsAction.Response> listener) {
+    protected void doExecute(Task task, PutSynonymsAction.Request request, ActionListener<SynonymUpdateResponse> listener) {
         synonymsManagementAPIService.putSynonymsSet(
```

Comment:
If we are not using `PutSynonymsAction.Response` class, should we delete it?

Reply:
Thanks for catching this! Removing